llvm/include/llvm/Object/Minidump.h

```diff
@@ -371,6 +377,14 @@ Expected<ArrayRef<T>> MinidumpFile::getDataSliceAs(ArrayRef<uint8_t> Data,
   return ArrayRef<T>(reinterpret_cast<const T *>(Slice->data()), Count);
 }
 
+// Needed by MinidumpTest.cpp
```

Comment:
Can we remove this? This is bound to get out of date over time. (In fact it's probably out of date already, as I suspect lldb needs this as well, but nobody is building it in a configuration where it matters).